docs/pages/build-reference/e2e-tests.md

```diff
@@ -357,3 +357,117 @@ Run a build with `eas build -p ios --profile test` and wait for the build to fin
 ## Repository
 
 The full example from this guide is available at https://github.com/expo/eas-tests-example.
+
+## Alternative Approaches
+
+### Using development builds to speed up test run time
+
+The above guide explains how to run E2E tests against a release build of your project, which requires executing a full native build before each test run. Re-building the native app each time you run E2E tests may not be desirable if only the project JavaScript or assets have changed, but this is necessary for release builds because the app JavaScript bundle is embedded into the binary.
+
+We can use [development builds](/development/introduction/) to load from a local development server or from [published updates](/eas-update/introduction/) instead, and save time and CI resources. This can be done by having your E2E test runner invoke the app with a URL that points to a specific update bundle URL, as described in the [development builds deep linking URLs guide](/development/development-workflows/#deep-linking-urls).
```

Comment:
```suggestion
Instead, we can use [development builds](/development/introduction/) to load from a local development server or from [published updates](/eas-update/introduction/) to save time and CI resources. This can be done by having your E2E test runner invoke the app with a URL that points to a specific update bundle URL, as described in the [development builds deep linking URLs guide](/development/development-workflows/#deep-linking-urls).
```